memory/memory_linux_test.go

```diff
@@ -78,7 +77,6 @@ DirectMap2M:      888832 kB
 		Buffers:    uint64(81744 * 1024),
```

Comment:
`expected.MemAvailableEnabled` should be `false` and currently it's tested implicitly by default value, but I think it's better declare `MemAvailableEnabled: false` explicitly for readability.

Reply:
👌